base_url/models/abstract_url.py

```diff
@@ -95,21 +106,28 @@ def set_url(self, url_key):
             ('redirect', '=', False)])
         redirect_urls.write({'redirect': True})
 
-    @api.depends('url_builder')
+    def _redirect_existing_url(self):
```

Comment:
@sebastienbeau This method is to process the existing url**s** `redirect_existing_urls`